.ci/run-repository.sh

```diff
@@ -68,11 +68,13 @@ elif [[ "$TASK_TYPE" == "doc" ]]; then
 elif [[ "$TASK_TYPE" == "trace" ]]; then
   # Set up OpenSearch cluster & Run model autotracing (Invoked by model_uploader.yml workflow)
   echo -e "\033[34;1mINFO:\033[0m MODEL_ID: ${MODEL_ID}\033[0m"
+  echo -e "\033[34;1mINFO:\033[0m MODEL_LICENSE: ${MODEL_LICENSE}\033[0m"
   echo -e "\033[34;1mINFO:\033[0m MODEL_VERSION: ${MODEL_VERSION}\033[0m"
   echo -e "\033[34;1mINFO:\033[0m TRACING_FORMAT: ${TRACING_FORMAT}\033[0m"
   echo -e "\033[34;1mINFO:\033[0m EMBEDDING_DIMENSION: ${EMBEDDING_DIMENSION:-N/A}\033[0m"
   echo -e "\033[34;1mINFO:\033[0m POOLING_MODE: ${POOLING_MODE:-N/A}\033[0m"
   echo -e "\033[34;1mINFO:\033[0m MODEL_DESCRIPTION: ${MODEL_DESCRIPTION:-N/A}\033[0m"
+  echo -e "\033[34;1mINFO:\033[0m MIT_LICENSE_URL: ${MIT_LICENSE_URL:-N/A}\033[0m"
```

Comment:
let's make this "Additional_LICENSE_URL"? In that way may be we can cover other use cases similar to MIT?

Reply:
Makes sense. Ack